styles/Canonical/003-Ubuntu-names-versions.yml

```
@@ -13,11 +13,11 @@ scope:
 level: suggestion
 tokens:
   # LTS releases should be followed by "LTS" suffix.
-  - 'Ubuntu (24|22|20|18|16|14|12|10|8)\.04(?! LTS)'
+  - 'Ubuntu (\d)?(0|2|4|6|8)\.04(?! LTS)'
   - 'Ubuntu 6\.06(?! LTS)'
   # Non-LTS releases should not be followed by "LTS" suffix.
-  - 'Ubuntu (23|22|21|20|19|18|17|16|15|14|13|12|11|10|9|8|7|6\5\4)\.10 LTS'
-  - 'Ubuntu (23|21|19|17|15|13|11|10|9|8|7|5)\.04 LTS'
+  - 'Ubuntu [\d]{1,2}\.10 LTS'
```

Isn't this more classic?

```suggestion
  - 'Ubuntu \d?\d\.10 LTS'
```

Or this more accurate

```suggestion
  - 'Ubuntu [1-9]?\d\.10 LTS'
```

As you don't want to allow 08.10

---

Well this is for disallowing those *.10 LTS's anyways, so we can even disallow that one